superset-frontend/src/explore/actions/datasourcesActions.ts

```diff
@@ -49,7 +49,7 @@ export function saveDataset({
       const {
         json: { data },
       } = await SupersetClient.post({
-        endpoint: '/superset/sqllab_viz/',
+        endpoint: '/api/v1/dataset/sqllab_viz/',
```

Comment:
This one will need be updated as well

Reply:
Also since this is explore, the response payload from `POST /api/v1/dataset/` maybe be missing keys that explore expects to be there in order to work properly. After this post we may need to make a request to `GET /api/v1/dataset/{id}`, which will return all needed keys.  See here in a recent PR of mine where I had to do something similar: https://github.com/apache/superset/pull/23678/files#diff-ccfa293ffd1a07eab82f8646fcb2732ff5d001fec273656b18064db5115ac9fbR165